docs/_specs/SPEC-009-cleanup-documentation.md

```
@@ -0,0 +1,259 @@
+kalshi data export --format parquet
+
+# Export to CSV
+kalshi data export --format csv
+```
+
+## Next Steps
+
+- Open `notebooks/exploration.ipynb` for interactive analysis
+- Read [USAGE.md](USAGE.md) for advanced features
+- Set up alerts for price movements
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)